drivers/device_uid/device_uid_handlers.c

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2018 Alexander Wachter
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <syscall_handler.h>
+#include <device_uid.h>
+
+Z_SYSCALL_HANDLER(device_uid_get, dev, buffer, length) {
+
+	Z_OOPS(Z_SYSCALL_DRIVER_DEVICE_UID(dev, device_uid_get));
+	Z_OOPS(Z_SYSCALL_MEMORY_WRITE(buffer, length));
```

Comment:
Does this still work for the case of `buffer` == NULL and `length` == 0?

Reply:
BTW, @alexanderwachter -- this doesn't seem like something that's going to work.